bindings/rust/Cargo.lock

```diff
@@ -0,0 +1,477 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This is an auto generated file.
We should not include it in source code repository.